### ISSUE TITLE
slider modules don't scroll well

### ISSUE THREAD

**Issue by ugoCapetoOfficial** [Alexays/Waybar#3249]
I have a slider for pulseaudio and backlight, both support scrolling, even though **it's not documented** in the man pages.
The scroll is buggy, it only works in one direction, and it's flickery, compared to the much better scroll provided in the pulseaudio and backlight modules. It would be great **to turn it off** or make it better.

**Comment by BIIJESH** [comment]
yes, this has been an issue after the recent update so i had to downgrade it
